src/pages/Signup/Signup.scss

```
@@ -1 +1,123 @@
 @import '../../styles/common.scss';
+
+// @mixin
+@mixin border-color {
+  border: 1px solid lightgray;
+  border-radius: 5px;
+}
+
+@mixin border-radius {
+  width: 300px;
+  height: 50px;
+  margin-bottom: 20px;
+  border-radius: 20px;
+}
+
+@mixin input-style {
+  width: 500px;
+  height: 50px;
+  padding: 10px;
+  margin-top: 5px;
+}
+
+@mixin headerFontSize {
+  color: gray;
+  font-size: 15px;
+}
+
+.Signup {
+  display: flex;
+  justify-content: center;
+
+  .signupView {
+    width: 600px;
+    margin-top: 200px;
+    flex-direction: column;
+
+    h1 {
+      margin-bottom: 20px;
+      font-size: 30px;
+    }
+
+    .signupName {
+      margin-top: 40px;
+
+      .signupHeader {
+        @include headerFontSize;
+      }
+
+      .signupInput {
+        @include border-color;
+        @include input-style;
+        margin-bottom: 20px;
+      }
+    }
+
+    .signupPhone {
+      .signupHeader {
+        @include headerFontSize;
+      }
+
+      .signupInput {
+        @include border-color;
+        @include input-style;
+        margin-bottom: 10px;
+      }
+    }
+
+    .signupDetail {
+      margin-top: 30px;
+
+      .signupHeader {
+        @include headerFontSize;
+      }
+
+      .signupInput {
+        @include border-color;
+        @include input-style;
+        margin-bottom: 20px;
+      }
+    }
```

```suggestion
    .signupInputWrapper {
      &:nth-of-type(1) {
        margin-top: 40px;
      }

      &:nth-of-type(2) {
        .signupInput {
          margin-bottom: 10px;
        }
      }

      &:nth-of-type(3) {
        margin-top: 30px;
      }

      .signupHeader {
        @include headerFontSize;
      }

      .signupInput {
        @include border-color;
        @include input-style;
        margin-bottom: 20px;
      }
    }
```
동일한 속성이 많이 겹쳐있는 것 같습니다!
위와 같이 간결하게 표현할 수 있습니다!

---

수정 완료!!! 이해 되지 않는 부분이 있어서 성훈님께 여쭤보고 수정했습니다!😃